docs/アイデア-2.md

```
@@ -0,0 +1,79 @@
+- 一番投票率が高かった人にベストアクター賞など称号がもらえる。
+
+### 勝利条件２
+- ゲームの勝敗と物語の勝敗の両方達成で完全勝利条件
+- ゲームの勝敗：村人陣営 or 人狼陣営が勝つ 
+- 物語の勝敗：個人が設定されたキャラ目標を達成できたかどうか  
+
+各プレイヤーは「役職」とは別に、キャラ設定や個人目標が与えられる。  
+この目標を達成すると、敗北しても「物語上の勝利」になる。
+
+例「忠実な執事（人狼）」 → 主人（指定されたキャラ）を最後まで守れれば追加勝利。  
+　「謎の旅人（村人）」 → 最後まで正体を隠し通す（＝自分の本当の設定をばらさない）と追加ポイント。
```

markdownlint-cli2 (0.18.1)

Files should end with a single newline character

(MD047, single-trailing-newline)